media/script/main.js

```diff
@@ -0,0 +1,54 @@
+const menuBars = document.querySelector('.menu-bars');
+const main = document.querySelector('.main-body');
+
+const menuDiv = document.createElement('div');
+menuDiv.className = 'toggle-nav';
+const menuUl = document.createElement('ul');
+const menuPortfolio = document.createElement('li');
+menuPortfolio.textContent = 'Portfolio';
+const menuAbout = document.createElement('li');
+menuAbout.textContent = 'About';
+const menuContact = document.createElement('li');
+menuContact.textContent = 'Contact';
+const closeBtn = document.createElement('span');
+closeBtn.textContent = 'x';
+closeBtn.id = 'toggle-close';
+closeBtn.style.color = 'white';
+
+menuUl.appendChild(menuPortfolio);
+menuUl.appendChild(menuAbout);
+menuUl.appendChild(menuContact);
+
+menuDiv.appendChild(closeBtn);
+menuDiv.appendChild(menuUl);
```

Comment:
[OPTIONAL] ♻
You've done a great job implementing the mobile menu feature, but be mindful that since your menuis dynamically generated through javascript, robots crawling the web for SEO purposes won't be able to render it and the result might be a poor ranking among search engines. Consider implementing the menu inside the main .html file and then manipulating it using JS!